.scala-steward.conf

```
@@ -1 +1,2 @@
 updates.ignore = [ { groupId = "io.sentry", artifactId = "sentry-logback" } ]
+updates.ignore = [ { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-databind", version="x.y." } ]
```

d'après la doc c'est `updates.pin` https://github.com/scala-steward-org/scala-steward/blob/master/docs/faq.md#how-can-version-updates-be-controlled
```suggestion
updates.pin = [ { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-databind", version="x.y." } ]
```

---

Bien vu 👍